File: kvt-projekat2024/src/app/models/AccountRequest.ts
export class AccountRequest {
    id: number;
    email: string;
    create_at: Date;
    address: string;
    status: RequestStatus;
    rejection_reason?: string;  
  
    constructor(id: number, email: string, create_at: Date, address: string, status: RequestStatus, rejection_reason?: string) {
      this.id = id;
      this.email = email;
      this.create_at = create_at;
      this.address = address;
      this.status = status;
      this.rejection_reason = rejection_reason;
    }
  }
  

File: svt-projekat2024/src/main/java/rs/ac/uns/ftn/svt/model/AccountRequest.java
package rs.ac.uns.ftn.svt.model;
import jakarta.persistence.*;
import lombok.Data;
import java.time.LocalDate;
import jakarta.validation.constraints.Email;

@Data
@Entity
public class AccountRequest {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    @Email(message = "Email should be valid")
    private String email;

    private LocalDate createdAt;

    private String address;

    @Enumerated(EnumType.STRING)
    private RequestStatus status;

    private String rejectionReason;
}
